docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -4,29 +4,23 @@ By [Phillip Carter](https://github.com/cartermp)
+Targeting .NET Core for libraries can be done entirely with the .NET CLI tools, which are a foundational set of tools used by Visual Studio and ASP.NET Core.  They provide an efficient and low-level experience that works across any supported OS.  You can still build libraries with Visual Studio, and if that is your preferred experience then you should [refer to the Visual Studio guide](libraries-with-vs.md).  This document will focus on using the CLI tools directly.
 
-* [Prerequisities](#prerequisites)
+* [Prerequisites](#prerequisites)
 * [How to target .NET Core](#how-to-target-net-core)
 * [How to target .NET Framework](#how-to-target-net-framework)
 * [How to target a Portable Class Library (PCL)](#how to target a portable class-library-pcl)
-* [How to cross-compile for .NET Core and .NET Framework](#how-to-cross-compile-for-net-core-and-net-framework)
+* [How to multitarget](#how-to-multitarget)
+* [How to use native dependencies](#how-to-use-native-dependencies)
 * [How to test libraries on .NET Core](#how-to-test-libraries-on-net-core)
+* [How to use multiple projects](#how-to-use-multiple-projects)
 * [How to create a NuGet Package with your Library](#how-to-create-a-nuget-package-with-your-library)
 
 ## Prerequisites
 
-You must have .NET Core installed on your machine.  You have two options:
+You must have .NET Core installed on your machine.  You will need [the .NET Core SDK and CLI](http://dotnet.github.io/getting-started/).
 
-* [DNX](http://aspnet.readthedocs.org/en/latest/getting-started/index.html), which is part of ASP.NET 5 RC1 and is included in Visual Studio Update 1.
-
-* [.NET CLI](http://dotnet.github.io/getting-started/), which will be used by .NET Core and ASP.NET RC2 and will replace DNX.
-
-Either toolchain can be used to build and publish libraries today.
-
-The sections of this document dealing with .NET Framework versions or Portable Class Libraries (PCLs) need the .NET Framework installed.  This is only supported on Windows.
-
-To do this, [install the .NET Framework](http://getdotnet.azurewebsites.net).
+The sections of this document dealing with .NET Framework versions or Portable Class Libraries (PCLs) need the .NET Framework installed.  They are only supported on Windows.  To do this, [install the .NET Framework](http://getdotnet.azurewebsites.net).
 
 Additionally, if you wish to support older targets, you will need to install targeting/developer packs for older framework versions from the [target platforms page](http://getdotnet.azurewebsites.net/target-dotnet-platforms.html).  Refer to this table:
 
```

Comment:
I only noticed this title "Thing to download" now. Maybe it would be better to say what to download or something like that.